DNS change clarification

MikeBazMSFT opened the pull request:
Following up on e-mail thread from June 26 around clarification of the DNS changes for geo-replication.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

MikeBazMSFT committed:
DNS change clarification

Following up on e-mail thread from June 26 around clarification of the DNS changes for geo-replication.
